### PR TITLE
Resized thumbnail feature

### PR DESCRIPTION
**Pull request by naofumi-fujii**
- switch thumbnail(resized or not) by `Site.resize_thumb[:boolean]`

### CODE REVIEW

**Comment by naofumi-fujii** [comment]
sorry, forgot to add some test
it fails currently... ☔️

**Comment by naofumi-fujii** [comment]
fixed 🔧

**Comment by naofumi-fujii** [comment]
cant see ci runnning on this PR... ?

File: db/structure.sql [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
 --
 
--- Dumped from database version 9.5.4
--- Dumped by pg_dump version 9.5.4
+-- Dumped from database version 9.6.1
+-- Dumped by pg_dump version 9.6.1
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```
Comment (naofumi-fujii) [inline db/structure.sql:10]: not sure about this change... 😟

Comment (tricknotes) [reply db/structure.sql:10]: Don't mind 😎

**Comment by naofumi-fujii** [comment]
>had a problem deploying to daimon-news-stg-pr-672

😵

File: db/migrate/20161220020802_add_column_to_site.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddColumnToSite < ActiveRecord::Migration
+  def change
+    add_column :sites, :resize_thumb, :boolean, default: false, null: false
```
Comment (mtsmfm) [inline db/migrate/20161220020802_add_column_to_site.rb:3]: only site?
we'll get higher benefit if it applies for `Post#thumbnail`

**Comment by naofumi-fujii** [comment]
>had a problem deploying to daimon-news-stg-pr-672

anything i can help?

**Comment by mtsmfm** [comment]
It seems that some thing get wrong on running postdeploy script.

Can you run postdeploy script locally?

https://github.com/bm-sms/daimon-news/blob/master/app.json#L61

File: db/migrate/20161220020802_add_column_to_site.rb (hunk 2 above)
Comment (mtsmfm) [reply db/migrate/20161220020802_add_column_to_site.rb:3]: Ah, this column will be not used I think because carrierwave doesn't store path for versions

**Review by mtsmfm — CHANGES_REQUESTED** [review]

File: db/migrate/20161220020802_add_column_to_site.rb (hunk 2 above)
Comment (naofumi-fujii) [reply db/migrate/20161220020802_add_column_to_site.rb:3]: this column was for something like on/off settings belongs to site, 
though i changed my mind to remove this and created new PR at #693

**Comment by naofumi-fujii** [comment]
closing due to 
https://github.com/bm-sms/daimon-news/pull/672#discussion_r96132694

**Pull request closed without merging**